module_auto_update/models/module.py

```diff
@@ -30,11 +30,15 @@ def _compute_checksum_dir(self):
         ).split(",")
 
         for r in self:
-            r.checksum_dir = dirhash(
-                get_module_path(r.name),
-                'sha1',
-                excluded_extensions=exclude,
-            )
+            try:
```

Comment:
This is here because `get_module_path` can return `False` if an addon is installed bus it not found on disk. Then, `dirhash` raises a `TypeError` because it expects a `str`.